src/components/AvatarToken.vue

```diff
@@ -1,11 +1,12 @@
 <script setup lang="ts">
 withDefaults(
   defineProps<{
-    src: string;
+    src?: string;
```

Comment:
This value seems to be used only internally by this component since the refactoring. Nobody ever passed `src` anymore, maybe just remove this?